### DISCUSSION TITLE
Help with UBL no probe

### DISCUSSION THREAD

**Discussion by fifo182** [Jyers/Marlin#478] [Q&A] [answered]
Hello. I'm still new to 3D printing. I downloaded the new firmware version 1.3.0 E3V2-UBL-NoProbe-5x5-v4.2.2.bin. I have no experience with this bed measurement. I used to use manual bed measurement. I had no problem with that. I don't know exactly how to proceed with UBL. Is there a step-by-step guide on how to proceed? I tried many times, but without success. Thank you for all the help.

**Comment by Jyers** [comment, marked as answer]
Unfortunately there is no detailed guide at the moment, but hopefully soon! It should be mostly the same process as manual, just using the manual section of the UBL menu.
